fitnessllm_dataplatform/entities/enums.py

```
@@ -0,0 +1,23 @@
+from enum import Enum
+
+
+class DynamicEnum(Enum):
+    """A blank enum that adds members via initialization function."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)